app/models/friendship.rb

```diff
@@ -4,4 +4,9 @@ class Friendship < ApplicationRecord
 
   belongs_to :user
   belongs_to :friend, class_name: 'User'
+
+  def confirmation
+    update_attributes(confirmed: true)
+    Friendship.create(user_id: friend_id, friend_id: user_id, confirmed: true)
+  end
```

Comment:
Use the following model as an example, add the self in front.

def confirm_friend
  self.update_attributes(confirmed: true)
  Friendship.create!(friend_id: self.user_id,
                  user_id: self.friend_id,
                  confirmed: true)
end

Reply:
Thank you @GzimAsani for this clear feedback. I agree with you. With the self, the code looks cleaner and is easier to follow.
However, rubocop doesn't allow it. When I run the linter check command, it tells me to remove the redundant self raising the errors in the screenshot below.
![rubocop error on redundant self](https://user-images.githubusercontent.com/44635784/125988057-08326fa1-5cb6-48be-a6a7-4624e54d6870.png)